student.rb

```diff
@@ -1,9 +1,17 @@
 require './person'
 
 class Student < Person
+  attr_reader :classroom, :rentals
+
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    @rentals = []
+  end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
   end
 
   def play_hooky
```

Comment:
### please these changes are For class `person `since you didn't make changes in it so i can't comment inside

- [ ] Note that A person has many rentals. kindly ensure @rental should be present as one of the instance variables
`attr_accessor :name, :age, :rentals`
- [ ] please ensure @rentals should be initialized as well
`@rentals = []`
- [ ] Lastly, kindly ensure a method for adding rentals should be present. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```